extensions/skins/conversation_v1/Conversation.js

```diff
@@ -285,6 +286,28 @@ oppia.directive('conversationSkin', [function() {
         }
       };
 
+      $scope.readSelectedTextAloud = function() {
+        var content = $scope.activeCard.contentHtml;
+        var contentElement = document.createElement('div');
+        contentElement.innerHTML = content;
+        var contentNodes = contentElement.childNodes[0].childNodes;
+        var stringToRead = "";
+        
+        for (var i = 0; i < contentNodes.length; i ++) {
+          var component = contentNodes[i];
+          stringToRead += speechService.getSpokenText(component);
+        };
+        var defaultMessage = stringToRead;
+        var selectedText =  $window.getSelection();
```

Comment:
I'm not sure this functionality is clear to the end user. Can we just read the entirety of the contentHtml for the time being?